pancancer-arch-3/scripts/kill_worker_daemon.sh

```diff
@@ -1,3 +0,0 @@
-#! /bin/bash
```

Comment:
I guess this script and `run_worker_daemon.sh` are now  a part of container-host-bag